server/images.go

```diff
@@ -60,12 +60,10 @@ func (m *Model) Prompt(request api.GenerateRequest) (string, error) {
 	}
 
 	var vars struct {
-		First  bool
```

Comment:
Unfortunately this will likely need to stay in because otherwise the template will error